server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -76,6 +79,10 @@ export class TenantManager {
             // if the tenant doesn't have key2, it will be empty string
             // we should fail token generated with empty string as key
             if (!tenantKeys.key2) {
+                Lumberjack.info(`Invalid key. Deleting from cache.`, lumberProperties);
+                this.cache.delete(tenantId).catch((error) => {
+                    Lumberjack.error(`Unable to delete key from cache.`, lumberProperties, error);
+                });
                 throw error1 instanceof jwt.TokenExpiredError
```

Comment:
Not related to this PR but we don't seem to be logging error1. I wonder if it will be useful, especially in the cases where it isn't an instance of TokenExpiredError. Same thing in line 101 too

Reply:
I agree. I can add logs for error1 and error2 in this pr